python/tirex_tracker/__init__.py

```diff
@@ -449,6 +450,18 @@ def _get_python_info(
         value=is_interactive,
     )
 
+    if ipython is None:
+        script_file_path = Path(extract_stack()[0].filename).resolve()
+        repo = git_repo_or_none(script_file_path)
+        if repo is not None and repo.working_tree_dir is not None:
+            script_file_path = script_file_path.relative_to(repo.working_tree_dir)
+        _add_python_result_entry(
+            results=results,
+            measure=Measure.PYTHON_SCRIPT_FILE_PATH,
+            measures=measures,
+            value=str(script_file_path),
+        )
+
```

Comment:
This is already done here:

https://github.com/tira-io/tirex-tracker/blob/5613bd9b827c5bd30b68afb3f78cee6172d36888/python/tirex_tracker/__init__.py#L458-L463

Reply:
I also do not understand why the script path should be resolved relative to the reository root dir and not the current working dir of the code starting the metadata tracking.

Reply:
Because the repository root is what is maintained in the metadata.

I.e., I do not know what script is executed otherwise.

Reply:
Okay, alright. I think I just do not yet understand the whole code zipping or why it uses Git. I'll create an issue on that matter, but that should not block this PR

Reply:
Discussion on that matter is continued in #48